src/command_modules/azure-cli-interactive/azclishell/_dump_commands.py

```diff
@@ -24,15 +24,18 @@
 
 def install_modules():
     for cmd in CMD_TABLE:
-        CMD_TABLE[cmd].load_arguments()
-
-    try:
+        try:
+            CMD_TABLE[cmd].load_arguments()
+        except (ImportError, ValueError):
+            pass
+        installed_command_modules = []
```

Comment:
Initialize this outside the for-loop as it's used in the for-loop below.
Would prevent a potential `'installed_command_modules' is not defined` error .